profiles/common/modules/custom/nexteuropa_json_field/nexteuropa_json_field.module

```diff
@@ -125,7 +125,8 @@ function nexteuropa_json_field_field_formatter_view($entity_type, $entity, $fiel
       foreach ($items as $delta => $item) {
         $element[$delta] = array();
         // Strip_tags to avoid javascript injection.
```

Comment:
As you removes strip_tags function, and uses filter_xss, can you also Update  this comment line?

Reply:
Comments updated